config/prod/arvados-wf-testing.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
+stack_name: arvados-wf-testing
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  SubProject: "arvados-trial"
```

Comment:
we don't support a `SubProject` parameter, please remove this.

Instead you can set `Project: "amp-ad/arvados-trial"`